vnext/Microsoft.ReactNative.Cxx/JSValueXaml.h

```diff
@@ -19,6 +19,53 @@ inline void ReadValue(JSValue const &jsValue, Windows::UI::Color &value) noexcep
   value = XamlHelper::ColorFrom([&jsValue](IJSValueWriter const &writer) noexcept { jsValue.WriteTo(writer); });
 }
 
+inline void ReadValue(JSValue const &jsValue, xaml::Thickness &value) noexcept {
```

Comment:
I think this is the first case that exposes conditional compilation between xaml versions . 
At the moment we only build and test standalone non winui3 of universal.
I'd suggest (future PR okay of course) adding an entry in the matrix of Universal for winui3 so we have UnitTest coverage of both conditional compilation paths of the `xaml::` namespace.

Reply:
#8358